wordpress-mobile/WordPress-iOS
# Quick Start v2: Add Ellipsis Menu

Refs #10860

This adds the ellipsis menu that allows quick start to be removed.

![ellipsis menu pr](https://user-images.githubusercontent.com/517257/52742131-f04c4280-2f8b-11e9-9eb4-0d1b7136cb37.gif)


To test:
- Don't force quick start to appear. You'll have to actually have QS enabled—putting a `QuickStartTourGuide.find()?.setup(for: blog)` somewhere makes this a big easier 😆
- Ensure that you can use the ellipsis menu to remove the QS cells
